scm/driver/github/content.go

```diff
@@ -55,20 +71,22 @@ type content struct {
 	Type    string `json:"type"`
 }
 
+type contentCreate struct {
+	Branch    string
+	Message   string
+	Content   []byte
+	SHA       string
+	Author    *scm.CommitAuthor
+	Committer *scm.CommitAuthor
+}
+
 type contentUpdate struct {
-	Sha     string `json:"sha"`
-	Message string `json:"message"`
-	HTMLURL string `json:"html_url"`
-	Author  struct {
-		Name  string    `json:"name"`
-		Email string    `json:"email"`
-		Date  time.Time `json:"date"`
-	} `json:"author"`
-	Committer struct {
-		Name  string    `json:"name"`
-		Email string    `json:"email"`
-		Date  time.Time `json:"date"`
-	} `json:"committer"`
+	Branch    string
+	Message   string
+	Content   []byte
+	SHA       string
+	Author    *scm.CommitAuthor
+	Committer *scm.CommitAuthor
```

Comment:
please revert these changes.  the `contentUpdate` structure should not rely on structures in the `scm` package.